README.md

```diff
@@ -422,6 +422,22 @@ but it will be deployed and utilized in the near future.
 Some improvements were made to the Topshot contract to reflect some cadence best practices and fix a bug.
 In-depth explanation on the changes and why we made them can be found in our [Blog Post](https://blog.nbatopshot.com/posts/nba-top-shot-smart-contract-improvements) 
 
+## TopShot Locking Contract Overview
+
```

Comment:
Add the name of the contract somewhere in here.

Also, mention some specific functions and transactions that can be used to lock/unlock moments and the process that the user should go through to do it.